sql/zone_settings.sql

```diff
@@ -234,7 +234,7 @@ INSERT INTO `zone_settings` VALUES (185,5,'127.0.0.1',54230,'Dynamis-San_dOria',
 INSERT INTO `zone_settings` VALUES (186,5,'127.0.0.1',54230,'Dynamis-Bastok',121,121,121,121,0,0.00,400);
 INSERT INTO `zone_settings` VALUES (187,5,'127.0.0.1',54230,'Dynamis-Windurst',121,121,121,121,0,0.00,400);
 INSERT INTO `zone_settings` VALUES (188,5,'127.0.0.1',54230,'Dynamis-Jeuno',121,121,121,121,0,0.00,400);
-INSERT INTO `zone_settings` VALUES (189,0,'127.0.0.1',0,'189',0,0,0,0,0,0.00,0);
```

Comment:
I am 1000% sure this is wrong. Not a typo. One thousand percent wrong. These do not use zone IDs per mog house.